### PR TITLE
chore(Everywhere): Various UI Tweaks, adds secondary-darker var

### PR DESCRIPTION
**Pull request by InfamousVague**
### What this PR does 📖

WIP

### CODE REVIEW

**Comment by InfamousVague** [comment]
Nothing major has been changed that needs testing, mainly just making sure nothing is broken (99% of it is CSS changes)

File: kit/src/components/message/mod.rs [hunk 1]
```diff
@@ -693,7 +693,16 @@ pub fn IdentityMessage(cx: Scope<IdentityMessageProps>) -> Element {
                     onpress: move |_| {
                         ch.send(IdentityCmd::SentFriendRequest(identity.did_key().to_string(), state.read().outgoing_fr_identities()));
                     },
-                    text: get_local_text_with_args("friends.add-name", vec![("name", identity.username())]),
+                    icon: if disabled {
+                        Icon::Check
```
Comment (stavares843) [inline kit/src/components/message/mod.rs]:
```suggestion
                        Icon::Check
```

Comment (stavares843) [reply kit/src/components/message/mod.rs]: linter was complaining about it

image

**Comment by InfamousVague** [comment]
ty for the fix @stavares843

**Comment by stavares843** [comment]
btw @InfamousVague we added this free bot for code reviews with AI but he isn't always right 😂 so if you see he's just talking trash to you just click in `resolve conversation`

Captura de ecrã 2023-11-09, às 01 48 43


 and all done 😂 

you can also add a PR comment to resolve all the comments at once


Captura de ecrã 2023-11-09, às 01 49 22

**Review by phillsatellite — APPROVED** [review]

File: kit/src/components/message/mod.rs [hunk 2]
```diff
@@ -702,7 +702,16 @@ pub fn IdentityMessage(cx: Scope<IdentityMessageProps>) -> Element {
...
+                    } else {
+                        Icon::Plus
+                    },
+                    text: if disabled {
+                        get_local_text("friends.already-friends")
+                    } else {
+                        get_local_text_with_args("friends.add-name", vec![("name", identity.username())])
+                    },
```
[the remark this answers is not in the document]
Comment (sdwoodbury) [reply kit/src/components/message/mod.rs:714]: another option is to create a variable above the rsx! macro. doing this inline certainly can be ugly.

**Review by sdwoodbury — APPROVED** [review]

**Pull request merged**